src/main/scala/com/databricks/spark/avro/AvroRelation.scala

```diff
@@ -117,6 +118,7 @@ private[avro] class AvroRelation(
    * contain `requiredColumns`
    */
   override def buildScan(requiredColumns: Array[String], inputs: Array[FileStatus]): RDD[Row] = {
+    val withalias = withAliasFields
```

Comment:
Is this just for serialization?  I'd probably just make sure to add transient elsewhere to match the style of other data sources.  Otherwise comment what is going on.